<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingHistory</class>
 <widget class="QWidget" name="TrainingHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="previous_Btn">
   <property name="geometry">
    <rect>
     <x>549</x>
     <y>824</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/release/source/image/left.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="next_Btn">
   <property name="geometry">
    <rect>
     <x>1072</x>
     <y>824</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/release/source/image/right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="page_Label">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>840</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1/1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>1620</width>
     <height>742</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border-radius: 8px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>1191</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget::item{  
    text-align:right;  
}</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditKeyPressed</set>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>1540</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #D8D8D8;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>108</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:  #E4F4FC;
border-radius: 25px;</string>
    </property>
    <property name="text">
     <string>时间</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>181</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:  #E4F4FC;
border-radius: 25px;</string>
    </property>
    <property name="text">
     <string>时长(min)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>108</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:  #E4F4FC;
border-radius: 25px;</string>
    </property>
    <property name="text">
     <string>日期</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>40</y>
      <width>165</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:  #E4F4FC;
border-radius: 25px;</string>
    </property>
    <property name="text">
     <string>运动模式</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>40</y>
      <width>108</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:  #E4F4FC;
border-radius: 25px;</string>
    </property>
    <property name="text">
     <string>得分</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="buttons_GroupBox">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>152</y>
      <width>311</width>
      <height>561</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="view0_Btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>38</y>
       <width>112</width>
       <height>68</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>112</width>
       <height>68</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background: #0D9DDB;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:hover{background: #0A90D4;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:pressed{background: #0984CE;
border-radius: 8px;
color: rgb(255, 255, 255);}</string>
     </property>
     <property name="text">
      <string>查看</string>
     </property>
    </widget>
    <widget class="QPushButton" name="import0_Btn">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>38</y>
       <width>112</width>
       <height>68</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>112</width>
       <height>68</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background: #0D9DDB;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:hover{background: #0A90D4;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:pressed{background: #0984CE;
border-radius: 8px;
color: rgb(255, 255, 255);}</string>
     </property>
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
    <widget class="QPushButton" name="view1_Btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>141</y>
       <width>112</width>
       <height>68</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>112</width>
       <height>68</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background: #0D9DDB;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:hover{background: #0A90D4;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:pressed{background: #0984CE;
border-radius: 8px;
color: rgb(255, 255, 255);}</string>
     </property>
     <property name="text">
      <string>查看</string>
     </property>
    </widget>
    <widget class="QPushButton" name="import1_Btn">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>141</y>
       <width>112</width>
       <height>68</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>112</width>
       <height>68</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background: #0D9DDB;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:hover{background: #0A90D4;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:pressed{background: #0984CE;
border-radius: 8px;
color: rgb(255, 255, 255);}</string>
     </property>
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
    <widget class="QPushButton" name="view2_Btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>244</y>
       <width>112</width>
       <height>68</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>112</width>
       <height>68</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background: #0D9DDB;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:hover{background: #0A90D4;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:pressed{background: #0984CE;
border-radius: 8px;
color: rgb(255, 255, 255);}</string>
     </property>
     <property name="text">
      <string>查看</string>
     </property>
    </widget>
    <widget class="QPushButton" name="import2_Btn">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>244</y>
       <width>112</width>
       <height>68</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>112</width>
       <height>68</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background: #0D9DDB;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:hover{background: #0A90D4;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:pressed{background: #0984CE;
border-radius: 8px;
color: rgb(255, 255, 255);}</string>
     </property>
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
    <widget class="QPushButton" name="view3_Btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>347</y>
       <width>112</width>
       <height>68</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>112</width>
       <height>68</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background: #0D9DDB;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:hover{background: #0A90D4;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:pressed{background: #0984CE;
border-radius: 8px;
color: rgb(255, 255, 255);}</string>
     </property>
     <property name="text">
      <string>查看</string>
     </property>
    </widget>
    <widget class="QPushButton" name="import3_Btn">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>347</y>
       <width>112</width>
       <height>68</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>112</width>
       <height>68</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background: #0D9DDB;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:hover{background: #0A90D4;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:pressed{background: #0984CE;
border-radius: 8px;
color: rgb(255, 255, 255);}</string>
     </property>
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
    <widget class="QPushButton" name="view4_Btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>450</y>
       <width>112</width>
       <height>68</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>112</width>
       <height>68</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background: #0D9DDB;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:hover{background: #0A90D4;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:pressed{background: #0984CE;
border-radius: 8px;
color: rgb(255, 255, 255);}</string>
     </property>
     <property name="text">
      <string>查看</string>
     </property>
    </widget>
    <widget class="QPushButton" name="import4_Btn">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>450</y>
       <width>112</width>
       <height>68</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>112</width>
       <height>68</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background: #0D9DDB;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:hover{background: #0A90D4;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:pressed{background: #0984CE;
border-radius: 8px;
color: rgb(255, 255, 255);}</string>
     </property>
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
